src/Pages/Nodes.razor

```diff
@@ -267,19 +272,55 @@
                 if (!result)
                 {
                     ToastService.ShowError($"{node.Name} could not be deleted");
+                    // IScheduler scheduler = await SchedulerFactory.GetScheduler();
+                    // scheduler.PauseJob(node.JobId.ToString());
+                    // scheduler.DeleteJob(node.JobId.ToString());
+                    
                 }
                 else
                 {
                     ToastService.ShowSuccess($"{node.Name} deleted successfully");
                     _nodes = await NodeRepository.GetAll();
 
                 }
+                
             }
         }
 
         await GetData();
 
     }
+    
+    private async Task CreateJobs(Node node)
+    {
+        try
+        {
+            IScheduler scheduler = await SchedulerFactory.GetScheduler();
+
+            var map = new JobDataMap();
+            map.Put("nodeId", node.Id);
+
+            var retryList = RetriableJob.ParseRetryListFromString(Constants.JOB_RETRY_INTERVAL_LIST_IN_MINUTES);
+            var job = RetriableJob.Create<NodeChannelSuscribeJob>(map, node.Id.ToString(), retryList);
+            await scheduler.ScheduleJob(job.Job, job.Trigger);
+
+            node.JobId = job.Job.Key.ToString();
+
+            var jobUpdateResult = NodeRepository.Update(node);
+
+            map = new JobDataMap();
+            map.Put("managedNodeId", node.Id.ToString());
+            
+            var acceptorJob = RetriableJob.Create<ProcessNodeChannelAcceptorJob>(map, node.Id.ToString(), retryList);
```

Comment:
do we need this? I think is already setup in Program.cs

Reply:
Done